.vib/odoo/cypress/cypress/fixtures/newUsers.json

```diff
@@ -0,0 +1,6 @@
+{
+  "newUser": {
+    "name": "Edward Rochester",
```

Comment:
It's a bit confusing having two fixture files so similar, but I think we are not using this `.name` in our tests.

Reply:
Indeed, I don't know what was I thinking when I did this! Fixed!